lib/drivers/docs/tandem.md

```diff
@@ -73,46 +76,50 @@ Device-specific? (Add any device-specific notes/additions here.)
 #### Device Events
 
   - alarms:
-    - `[?]` low insulin
-    - `[?]` no insulin
-        - `[?]` needed to infer a suspend (stoppage of all insulin delivery)
-    - `[?]` low power
-    - `[?]` no power
-        - `[?]` needed to infer a suspend (stoppage of all insulin delivery)
-    - `[?]` occlusion
+    - `[x]` low insulin
+    - `[x]` no insulin
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[x]` low power
+    - `[x]` no power
         - `[?]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[x]` occlusion
+        - `[x]` needed to infer a suspend (stoppage of all insulin delivery)
     - `[?]` no delivery
         - `[?]` needed to infer a suspend (stoppage of all insulin delivery)
```

Comment:
Maybe include a note about the `?`s here for `no delivery`? If my memory serves, there just isn't any alarm type that straightforwardly maps to `no delivery` for Tandem (which is fine but good to note).